.github/workflows/ci.yaml

```diff
@@ -41,6 +41,7 @@ jobs:
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: ${{ matrix.ruby-version }}
+        bundler: 2.3.18
```

Comment:
NOTE I don't know that this is a good idea, if nothing else we'd have to blast it out to all the repos

Reply:
😭

Reply:
If we are going to pay the price of blasting it out, I'd prefer we were blasting out something that we liked.

This feels like a hack.
And then blasting makes it feel even more like a hack.

Do we know why this is an issue?

Reply:
Some of this is coming back to me.
Do we know their solution for having multiple gem sources?

Reply:
Yeah I strongly dislike this, we need to report the bug to bundler and maybe make this change only in cross-repo where it is occurring

Reply:
> Do we know why this is an issue?

We only know that this is a problem with bundler 2.3.19 with cross repo which we've only been able to recreate on cross repo locally like this:

```
CI=true REPOS="ManageIQ/manageiq#22008,ManageIQ/manageiq-ui-classic#8373" TEST_REPO="ManageIQ/manageiq-ui-classic#8373" bundle exec manageiq-cross_repo
```

I know there are at least one other failing PR in cross repo tests that would likely have the same error.

It's interesting it's happening in the `bin/setup` after bundler is done and when it's ["Resettings tests"](https://github.com/ManageIQ/manageiq/blob/master/bin/setup#L37)

Reply:
![image](https://user-images.githubusercontent.com/19339/181763998-ca1a30f7-683f-41a1-bcdc-6371c456673f.png)

Reply:
> Yeah I strongly dislike this, we need to report the bug to bundler and maybe make this change only in cross-repo where it is occurring

I'm fine with forcing bundle 2.3.18 on cross repo tests until we figure it out.  That at least limits the scope of the change.

Reply:
Using bundler 2.3.18 on cross-repo resolves the issue, closing this